docs/concepts/nodes.md

```diff
@@ -84,21 +75,23 @@ Both Go-IPFS and JS-IPFS nodes use bootstrap nodes to initially enter the DHT.
 Features of a bootstrap node:
 
 - All default bootstrap nodes are part of the public DHT.
-- They are used by both Go-IPFS and JS-IPFS nodes.
-- The list of bootstrap nodes a Go-IPFS or JS-IPFS node connects to is configurable.
+- The list of bootstrap nodes a Go-IPFS or JS-IPFS node connects to is configurable in their config files.
 
 Limitations of a bootstrap node:
 
 - If an IPFS node only has one bootstrap node listed in that configuration and that bootstrap node goes offline, the IPFS node will lose access to the public DHT if it were to restart.
 
-### Delegate routing
+[More about Bootstrapping](../how-to/modify-bootstrap-list.md)
+
+### Delegate routing node
 
-When IPFS nodes are unable to run DHT logic on their own, they _delegate_ the task to a delegate routing node. Publishing works with arbitrary CID codecs, as the [js-delegate-content module](https://github.com/libp2p/js-libp2p-delegated-content-routing/blob/master/src/index.js#L127-L128) publishes CIDs at the block level rather than the IPLD or DAG level.
+When IPFS nodes are unable to run Distributed Hash Tag (DHT) logic on their own, they _delegate_ the task to a delegate routing node.  Publishing works with arbitrary CID codecs (compression/decompression technology), as the [js-delegate-content module](https://github.com/libp2p/js-libp2p-delegated-content-routing/blob/master/src/index.js#L127-L128) publishes CIDs at the block level rather than the IPLD or DAG level.
```

Comment:
```suggestion
When IPFS nodes are unable to run Distributed Hash Table (DHT) logic on their own, they _delegate_ the task to a delegate routing node.  Publishing works with arbitrary CID codecs (compression/decompression technology), as the [js-delegate-content module](https://github.com/libp2p/js-libp2p-delegated-content-routing/blob/master/src/index.js#L127-L128) publishes CIDs at the block level rather than the IPLD or DAG level.
```